x/upgrade/go.mod

```diff
@@ -180,4 +180,9 @@ require (
 
 // Fix upstream GHSA-h395-qcrw-5vmq and GHSA-3vp4-m3rf-835h vulnerabilities.
 // TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
-replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.9.1
+
+replace (
+	cosmossdk.io/core => ../../core
+	github.com/cosmos/cosmos-sdk => ../../.
```

Comment:
Why is this needed?

Reply:
upgrade still need use [sdk.ResponsePreBlock](https://github.com/cosmos/cosmos-sdk/blob/f3d2aa6c0469285ab53835518f79122a0cd06da9/x/upgrade/abci.go#L25)